arm-storage/2016-05-01/swagger/storage.json

```diff
@@ -0,0 +1,1327 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "StorageManagementClient",
+    "description": "The Storage Management Client.",
+    "version": "2016-05-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json",
+    "text/json"
+  ],
+  "produces": [
+    "application/json",
+    "text/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Storage/checkNameAvailability": {
+      "post": {
+        "tags": [
+          "StorageAccounts"
+        ],
+        "operationId": "StorageAccounts_CheckNameAvailability",
+        "description": "Checks that the storage account name is valid and is not already in use.",
+        "parameters": [
+          {
+            "name": "accountName",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/StorageAccountCheckNameAvailabilityParameters"
+            },
```

Comment:
Yes. Only two new features are added compared to previous version. Other features are same to previous version.
https://github.com/Azure/azure-rest-api-specs/pull/742/files#diff-5b8b9619ecdc1dedc0bfccf4e2662d8aR417
https://github.com/Azure/azure-rest-api-specs/pull/742/files#diff-5b8b9619ecdc1dedc0bfccf4e2662d8aR467

Once an api version is released(say 2016-01-01), feature team would like to working on new features to a new api version instead of adding things to a released version. That is also why we update the api version. I have an initial commit to copy swagger from 2016-01-01 to 2016-05-01, you can compare changes to that commit to see newly added changes.
https://github.com/Azure/azure-rest-api-specs/pull/742/commits/89e216ea837a23c5a77102d340c78d6563e98889